utils/generate_coverage

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+HARE_BASE_DIR=/opt/seagate/cortx/hare
+PYTHONPATH="$HARE_BASE_DIR/lib/python3.6/site-packages"
+PYTHONPATH="$HARE_BASE_DIR/lib64/python3.6/site-packages:$PYTHONPATH"
+export PYTHONPATH
+		
+PATH="$HARE_BASE_DIR/bin:$PATH"
+export PATH
+
+currentdir_path=`pwd`
+curren_dir_name=${currentdir_path##*/}
+
+check_hare_home() {
+    if [[ $curren_dir_name != 'hare' ]]; then
+        echo "Please run the script from Hare home directory."
+        exit 1
+    fi
+}
+activate () {
+    . .py3venv/bin/activate
+}
+
+check_root () {
+  if [[ $EUID -ne 0 ]]; then
```

Comment:
You're mixing tabwidth (2 spaces here, but 4 spaces above). Let's use 4.

Reply:
Removed check_root  as it is not needed.